src/content/components/TreeView.js

```diff
@@ -157,38 +173,36 @@ class TreeViewRowScrolledColumns extends PureComponent {
   }
 }
 
-TreeViewRowScrolledColumns.propTypes = {
-  node: PropTypes.object.isRequired,
-  nodeId: PropTypes.number.isRequired,
-  depth: PropTypes.number.isRequired,
-  mainColumn: PropTypes.shape({
-    propName: PropTypes.string.isRequired,
-    title: PropTypes.string.isRequired,
-  }).isRequired,
-  appendageColumn: PropTypes.shape({
-    propName: PropTypes.string.isRequired,
-    title: PropTypes.string.isRequired,
-  }),
-  appendageButtons: PropTypes.arrayOf(PropTypes.string),
-  index: PropTypes.number.isRequired,
-  canBeExpanded: PropTypes.bool.isRequired,
-  isExpanded: PropTypes.bool.isRequired,
-  onToggle: PropTypes.func.isRequired,
-  selected: PropTypes.bool.isRequired,
-  onClick: PropTypes.func.isRequired,
-  onAppendageButtonClick: PropTypes.func,
-  highlightString: PropTypes.string,
+type TreeViewPropsBase = {
+  fixedColumns: Column[],
+  mainColumn: Column,
+  tree: ProfileTreeClass,
+  expandedNodeIds: Array<IndexIntoFuncStackTable | null>,
+  selectedNodeId: IndexIntoFuncStackTable | null,
+  onExpandedNodesChange: PropTypes.func.isRequired,
+  highlightString: string,
+  appendageColumn: Column,
+  appendageButtons: string[],
+  disableOverscan: boolean,
+  icons: IconWithClassName[],
+  onAppendageButtonClick: ((IndexIntoFuncStackTable | null, string) => any) | null,
+  onSelectionChange: IndexIntoFuncStackTable => any,
 };
 
+type TreeViewProps = TreeViewPropsBase & ({ contextMenu: any } | { contextMenuId: string });
```

Comment:
Also on these I'd try to remove the `any`s with voids, and try for `React$Component<*>` in the case of the `contextMenu` if that passes.